testlapack/dbdsqrbench.go

```diff
@@ -0,0 +1,88 @@
+// Copyright ©2016 The gonum Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style
+// license that can be found in the LICENSE file.
+
+package testlapack
+
+import (
+	"math/rand"
+	"testing"
+
+	"github.com/gonum/blas"
+)
+
+type DbdsqrerDgebrder interface {
```

Comment:
If `Dgebrd` (and therefore this interface) is really necessary, then call it `DbdsqrDgebrder`?